podman/tests/unit/test_containers.py

```diff
@@ -514,9 +509,9 @@ def test_top(self, mock):
                     '/usr/bin/ssh-agent /bin/sh -c exec -l /bin/bash -c "/usr/bin/gnome-session"',
                 ],
                 ['jhonce', '5544', '3522', '0', 'Mar01', 'pts/1', '00:00:02', '-bash'],
-                ['jhonce', '6140', '3522', '0,''Mar01', 'pts/2', '00:00:00,''-bash'],
+                ['jhonce', '6140', '3522', '0,' 'Mar01', 'pts/2', '00:00:00', '-bash'],
```

Comment:
I think you want this, I don't see this being a test for a badly formatted array
```suggestion
                ['jhonce', '6140', '3522', '0', 'Mar01', 'pts/2', '00:00:00', '-bash'],
```